app/controllers/taxons_controller.rb

```diff
@@ -21,7 +21,7 @@ def redirect_to_www
   def taxon
     @taxon ||= begin
       taxon_base_path = params[:taxon_base_path]
-      Taxon.find('/alpha-taxonomy/' + taxon_base_path)
+      Taxon.find("/" + taxon_base_path)
```

Comment:
content-store solves the confusion by calling it `base_path_without_root`:

https://github.com/alphagov/content-store/blob/master/config/routes.rb#L4 

We can also use `request.path` like topics controller:

https://github.com/alphagov/collections/blob/master/app/controllers/topics_controller.rb#L3